pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -1073,12 +1070,13 @@ func (vs *VSphere) DisksAreAttached(volPaths []string, nodeName k8stypes.NodeNam
 		}
 
 		if !nodeExist {
-			glog.Errorf("DisksAreAttached failed to determine whether disks %v are still attached: node %q does not exist",
-				volPaths,
-				vSphereInstance)
-			return nil, fmt.Errorf("DisksAreAttached failed to determine whether disks %v are still attached: node %q does not exist",
-				volPaths,
-				vSphereInstance)
+			glog.Infof("Node %q does not exist, all the disks on this node will be detached automatically.", nodeName)
```

Comment:
During host failover when node VM goes to disconnected state or is not accessible for any reason we are returning a Map of volumepath and its attachment status set to false. This will be misinterpreted as disks are already detached from the node and Kubernetes mark volumes as detached after orphaned pod is cleaned up. 

This can causes volumes to remain attached to previous node, and new pod creation on another node will remains in the “containercreating” state. Since both the node vms are powered on and hot volumes can not be attached to other node.

We need to test this PR change for vSphere HA failover.

@BaluDontu @luomiao @tusharnt @rohitjogvmw